tests/test_config.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+import shutil
+from configparser import ConfigParser
+
+from aw_core import dirs
+from aw_core.config import load_config, save_config
+
+class ConfigTest(unittest.TestCase):
```

Comment:
There's no need to use the class structure with pytest. You're probably better off only having a stand-alone test function.